website/blog/2023-12-06-release-1.6.md

```diff
@@ -0,0 +1,250 @@
+---
+title: Release Notes - Podman Desktop 1.6
+description: Podman Desktop 1.6 has been released!
+slug: podman-desktop-release-1.6
+authors: slemeur
+tags: [podman-desktop, release, kubernetes, openshift, onboarding, compose, extensions, settings]
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.6/SantaSeal.png
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.6 Release! 🎉
+
+![Podman-desktop-1-6-hero](img/podman-desktop-release-1.6/SantaSeal.png)
+
+This release introduces:
+- **Minikube featured extension**: Minikube extension to create local Kubernetes clusters in containers.
+- **Podman 4.8.2**: [Podman 4.8.2](https://github.com/containers/podman/releases) is now included in Windows and Mac installers.
+- **Setting page for Command-Line tools**: Manage and update your CLI tools.
+- **Kubernetes contexts Manager**: Browse all your kubernetes context, set default and remove unused ones.
+- **Editable Podman Machine**: Easy resize and reconfiguration of the Podman runtime environment.
+- **Filters for Containers and Pods lists**: Focus on the containers and Pods you are working with.
+- **Sorting on Volumes and Images list**: Sort volumes or images with your prefered criterias.
+- **Enhancement to show environment colums on Containers and Pods lists**: 
+- **Extension API improvements**: Another set of improvements to the extension API enabling more goodness from 🦭 Podman Desktop's extensions
+
+Podman Desktop 1.6 is now available. [Click here to download it](/downloads)!
+
+---
+
+## Release Details
+
+### Minikube featured extension
+For developers who need to run Kubernetes locally and reproduce an environment close to production for development and experimentation purposes, Podman Desktop allows users to easily set up that environment on a local machine. There are two distincts extensions providing the capability to configure a Kubernetes cluster locally, you can either choose between [Kind](https://kind.sigs.k8s.io/) or [minikube](https://minikube.sigs.k8s.io/docs/).
```

Comment:
'distincts' -> 'distinct', or maybe remove entirely?

It also feels like you're excluding OpenShift Local or even Sandbox - maybe this is 'two built-in extensions' or 'configuring an open source Kubernetes' just to distinguish a bit?